features/interview/config/loader.py

```
@@ -39,10 +39,22 @@ def _load_stages_yaml() -> StagesConfig:
     """YAML 파일 로드 (캐싱)"""
     yaml_path = Path(__file__).parent / "stages.yaml"
 
-    with open(yaml_path, encoding="utf-8") as f:
-        data = yaml.safe_load(f)
+    try:
+        with open(yaml_path, encoding="utf-8") as f:
+            data = yaml.safe_load(f)
+    except OSError as exc:
+        raise ValueError(f"인터뷰 설정 파일을 읽을 수 없습니다: {exc}") from exc
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)